Library/Formula/testdisk.rb

```diff
@@ -1,13 +1,17 @@
-require 'formula'
-
 class Testdisk < Formula
-  homepage 'http://www.cgsecurity.org/wiki/TestDisk'
-  url 'http://www.cgsecurity.org/testdisk-6.14.tar.bz2'
-  sha1 'a2359406db5e3e3f9db81d8e8fce2a125dcc3677'
+  homepage "http://www.cgsecurity.org/wiki/TestDisk"
+  url "http://www.cgsecurity.org/testdisk-7.0.tar.bz2"
+  sha256 "00bb3b6b22e6aba88580eeb887037aef026968c21a87b5f906c6652cbee3442d"
 
   def install
-    system "./configure", "--disable-dependency-tracking",
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
                           "--prefix=#{prefix}"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/testdisk", "/help"
```

Comment:
I knew you'd say that. :)

Not sure what I can do for a test here except create a little disk image with hdiutil and then get test disk to "/list" the structure. Would that be okay?